Jenkinsfile

```diff
@@ -89,4 +98,7 @@ stage('Coverage') {
 
 stage('Done') {
         echo 'Done! :)'
+	slackSend channel: channel_name, color: 'good', message: "CI Tests finished! ${env.JOB_NAME} (<${env.BUILD_URL}|Open>)", teamDomain: 'raccoongang', tokenCredentialId: 'slack-secret-token'
```

Comment:
this is our slack creds or it's just a stub?

Reply:
yes, we have channel called "ci-open-edx"